tasks/styles.js

```diff
@@ -45,28 +45,23 @@ module.exports = (gulp, $, pkg) => {
         importer: $.magicImporter({
           disableImportOnce: true
         }),
-        outputStyle: options.outputStyle
```

Comment:
Why did you drop this?

Reply:
I was used by the `sass` plugin to create a minified build. The problem is: there's no way to get the minified and un-minified version of the file when using the `sass` plugin for minification – it would have to run twice.
Instead, I reused the `cleanCss` plugin (already included), which runs on top of the compiled css and also results in a (marginally) smaller file size.

Reply:
So `outputStyle` was actually just used to control minification, which now is (somewhat straightforward) called `minify`.